engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -909,7 +958,9 @@ namespace dmGameSystem
         SpriteContext* sprite_context = (SpriteContext*)params.m_Context;
         SpriteWorld* sprite_world = (SpriteWorld*)params.m_World;
 
-        UpdateTransforms(sprite_world, sprite_context->m_Subpixels);
+        UpdateTransforms(sprite_world, sprite_context->m_Subpixels); // TODO: Why is this not in the update function?
+
+        CalcBoundingVolumes(sprite_world); // uses the m_World to calculate the actual radius of the object
```

Comment:
I suspect that the UpdateTransform is done here, in order to compensate for the Spine component being able to move attached sprites.
We should be able to move this once the Spine component _before_ the sprite component:
[extension-spine](https://github.com/defold/extension-spine/blob/main/defold-spine/src/comp_spine_model.cpp#L1138-L1141)